python/smqtk/algorithms/relevancy_index/libsvm_hik.py

```diff
@@ -400,22 +421,25 @@ def rank(self, pos, neg):
             probs = 1.0 / (1.0 + numpy.exp((margins - rho) * probA + probB))
 
             # Detect whether we need to flip probabilities
+            # (similar to check on L367)
             # - Probability of input positive examples should have a high
-            #   probability score among the generated probabilities of our index.
-            # - If the positive example probabilities show to be in the lower 50%,
-            #   flip the generated probabilities, since its experimentally known
-            #   that the SVM will change which index it uses to represent a
-            #   particular class label occasionally, which influences the Platt
-            #   scaling apparently.
+            #   probability score among the generated probabilities of our
+            #   index.
+            # - If the positive example probabilities show to be in the lower
+            #   50%, flip the generated probabilities, since its experimentally
+            #   known that the SVM will change which index it uses to represent
+            #   a particular class label occasionally, which influences the
+            #   Platt scaling apparently.
             pos_vectors = numpy.array(train_vectors[:num_pos])
             pos_test_k = compute_distance_matrix(svm_SVs, pos_vectors,
                                                  histogram_intersection_distance,
                                                  row_wise=True)
             pos_margins = numpy.dot(weights, pos_test_k)
             #: :type: numpy.core.multiarray.ndarray
             pos_probs = 1.0 / (1.0 + numpy.exp((pos_margins - rho) * probA + probB))
-            # Check if average positive probability is less than the average index
-            # probability. If so, the platt scaling probably needs to be flipped.
+            # Check if average positive probability is less than the average
+            # index probability. If so, the platt scaling probably needs to be
+            # flipped.
             if (pos_probs.sum() / pos_probs.size) < (probs.sum() / probs.size):
                 self._log.debug("inverting probabilities")
                 probs = 1. - probs
```

Comment:
# Getting feedback using LIBSVM
```
_, feedback_pool = self.get_rank_and_feedback_using_libsvm(svm_model)
```
We want to get the feedback even when we're not using `use_libsvm = True`?

Reply:
That should probably be separate PR given the scope of that change.  Additionally, this was not happening before, so I did not change that behavior.